docs/tutorials/AddRemark.md

```diff
@@ -1,34 +1,37 @@
 ---
-layout: page
-title: "Tutorial: Adding a command"
+layout: page title: "Tutorial: Adding a command"
 ---
 
-Let's walk you through the implementation of a new command — `remark`.
+Let's walk you through the implementation of a new command — `remark`.
 
-This command allows users of the AddressBook application to add optional remarks to people in their address book and edit it if required. The command should have the following format:
+This command allows users of the AddressBook application to add optional remarks to people in their address book and
+edit it if required. The command should have the following format:
```

Comment:
Again, the auto formatting messing things up. Not impt here tho cuz this is the tutorial/add-remark part.